README.md

```diff
@@ -1 +1,109 @@
-# angular-concepts-readme
+# Angular Core Concepts
+
+## Overview
+
+In this lesson, we are going to look at the core concepts that we will use in every Angular application. Some will be familiar to you, however some may seem a bit strange. Don't worry if they take time to sink in!
+
+## Objectives
+
+- Define MVC (Model-View-Controller), MVVM (Model-View-ViewModel)
+- Describe one-way and two-way data-binding
+- Describe Angular's two-way binding
+- Describe Dependency Injection
+- Describe a Single Page Application and common features
+- Demonstrate server-side rendering concepts
+
+## Single Page Applications
+
+Angular allows us to create single page applications (SPAs). The means that the user goes to one page and very rarely will have to refresh/leave that page to go to other parts of the website.
+
+Single page applications handle all the routing, calling different parts of your codebase dependant on what URL the user is currently at.
+
+Tranditionally, server-side rendering would be like the following -
+
+![Server-side rendering](http://www.michaelgallego.fr/images/posts/2012-11-26-client-side-1.png)
+
+The user requests the page from the server, and the server will return all the HTML to display on that page. The data would not be updated until the user refreshes. This puts overhead on the server, as it will have to render the page on every page load.
```

Comment:
Nice explanation of server side rendering here! Can you incorporate the example below somehow? IE if 1000 users go to this website, the server is rendering 1000 different finished HTML templates.